### ISSUE TITLE
Time reductions from Easter event causing delays

### ISSUE THREAD

**Issue by A69Tor** [MatthieuBonne/CQAutomater#4]
Due to the time reduction rewards from the Easter event, Miracle and PVP battle timers are off, causing less than optimal auto-PVP and miracle collection.

**Comment by MatthieuBonne** [comment]
Just checked, timers look fine to me, playfab returns timestamps that include the bonus. Can you please send more info ?

**Comment by A69Tor** [comment]
Looks like I was mistaken on the "Miracles" timer, but I am seeing that my PVP timer is showing 40 minutes left, but the website is showing 33 minutes, even after multiple refreshes and restarts.  Is a timestamp received for PVP as well?  I can attach SS if that with help, looks to be about 9 minutes off on Automater side.

**Comment by MatthieuBonne** [comment]
Can you please test latest version (4.3.0.4) and tell me if you see some improvement ?

**Comment by Vitae666** [comment]
![image](https://user-images.githubusercontent.com/9310914/61586115-6efc6b80-ab3a-11e9-87eb-79dd3feb4ccf.png)
ATor69 isn't wrong about the timers being off. If I close Automater and reopen it, the time fixes. I eventually start gaining attacks. Right now I have 2 avail when I should have 0. I cleared it out yesterday.
Added note: It says the battle would be ready @ in 2 minutes @ 10:06:11 PM but it already was 10:06PM

**Comment by Vitae666** [comment]
![image](https://user-images.githubusercontent.com/9310914/61676226-28894700-acc9-11e9-8718-504dff9377a3.png)

**Comment by Vitae666** [comment]
Ok, correction to my previous "the time fixes" cause I just closed it and restarted it.
![image](https://user-images.githubusercontent.com/9310914/61676380-a8afac80-acc9-11e9-8a9c-85d134bd2c1a.png)
It did 1 attack. 

Ok, so I just unchecked "Enable Auto-PvP, waited a few seconds, and rechecked it.
The timer changed to match the game, did an attack and immediately reset the countdown to 1:00:05

BTW, can this get set to know how many attacks you have and use them all?
